CONTIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+This project is still at the initial stage of development. Feel free to submit PR/issue for any feedbacks.
```

Comment:
I'm not quite sure how true "still at the initial stage of development" is. While we might like to make some major (breaking) changes for v2 of the library there are quite a few projects that already import `github.com/go-ping/ping ` in 'production' as it were, as evidenced by [pkg.go.dev](https://pkg.go.dev/github.com/go-ping/ping?tab=importedby) and even some that aren't on there e.g. [smokeping_prober].(https://github.com/SuperQ/smokeping_prober).

Reply:
Thanks for the feedbacks! Just updated the fix. I've also noticed this PR[1]. It seems that we don't have a conclusion yet. I only added a reminder for the contributors to know that this project is used in production services. I think we might need to discuss it sooner or later. I think @SuperQ @sparrc both wanted the best for the community. Maybe we can just tag the current(legacy) version to `v1.0.0`. And we can start breaking and testing things in `v2.0.0`, since `v2` is not compatible with v1. This way, we can happily do any improvement we want and update README for new users to use `v2`. And old users can continue to use v1's API. 😄 

[1] https://github.com/go-ping/ping/pull/101